incubator-tools/cdc_comparison/cdc_comparison.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #124.            try:
</pre><p>These really shouldn't be try/excepts. These can be written something like this:</p><p> </p><pre class="ql-syntax" spellcheck="false">GT_type = getattr(GT_ent, "type", None)
</pre><p>Or this</p><pre class="ql-syntax" spellcheck="false">GT_type = GT_ent.type if GT_ent.type else None
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Updated